packages/react-core/src/components/Hint/examples/Hint.md

```diff
@@ -20,3 +20,15 @@ import EllipsisVIcon from '@patternfly/react-icons/dist/esm/icons/ellipsis-v-ico
 ```ts file="HintBasicWithoutTitle.tsx"
 
 ```
+
+### With actions
+
+You can add actions to a hint to allow users to respond to suggestions and initiate any related processes.
+
+When a `<Hint>` contains `actions`, a negative margin is applied to the actions for proper alignment between the hint title and the actions. For larger-sized titles or non-plain actions such as a button, remove this negative margin by setting `hasNoActionsOffset` to `true`.
```

Comment:
```suggestion
When a `<Hint>` contains `actions`, a negative margin is applied to the actions so that they are better aligned with the hint title. For larger-sized titles or non-plain actions (such as a button), remove this negative margin by setting `hasNoActionsOffset` to `true`.
``` 
nitpicky but maybe this so there's one less "actions"?

Reply:
Wow yes thanks, reading it back now all i see is the word "actions" so this change is worth it 👏